src/dags/aries_task_queuer.py

```diff
@@ -37,7 +37,7 @@
 ) as dag:
     aries_collector = TaskRequestBatchSensor(
         task_id=TASK_ID_REQUEST_SENSOR,
-        max_requests=max_requests,
+        max_requests=50,
```

Comment:
@joelkbiju12 always use variables instead of numbers directly.

Reply:
Will make the change and fetch the value directly from airflow cfg in helm chart. This is for testing purpose as I cannot directly edit the production helm chart